src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -708,8 +739,18 @@ private void AnalyzeString(string strValue, Statistics stat)
         /// </summary>
         private void AnalyzeNumber(double numValue, Statistics stat)
         {
-            if (_measureSum || _measureAverage)
+            if (_measureSum || _measureAverage || _measureStdDeviation)
+            {
+                stat.sumPrevious = stat.sum;
                 stat.sum += numValue;
+            }
+            if (_measureStdDeviation && stat.count > 1)
+            {
```

Comment:
let's leave a link in the comment to Wikipedia article the interative formula calculation - would be easier to reason in the future about this non-trivial part.

Reply:
I agree - I believe the public comment above is good place.